scenes/src/app/scenes/divider/divider-scene.scss

```diff
@@ -0,0 +1,23 @@
+app-divider-scene .mat-list .mat-list-item {
+  font-size: 40px;
+  height: 130px;
```

Comment:
No need to mess with the dimensions on everything, just use `transform: scale()` (Does the scene viewer have built in support for this? I remember discussing it